week1/index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="ko">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css">
+  <title>🍔 햄식이를 도와줘 🍔 - 김형겸</title>
+</head>
+
+<body>
+  <header>
+    <h1>🍔 햄식이를 도와줘 🍔</h1>
+  </header>
+  <main>git
+    <aside class="sidebar">
+      <h2 class="sidebar-title">다른거 먹자</h2>
+      <div class="sidebar-button">
+        <button class="option">맥도날드</button>
+        <button class="option">버거킹</button>
+      </div>
+    </aside>
```

Comment:
저도 BEM을 잘 아는 건 아니지만 .. .
여기서 '-'로 명시해주신 부분이 element단위인 거 같아서 '__'가 조금 더 적절할 것 같아요! 
```suggestion
    <aside class="sidebar">
      <h2 class="sidebar__title">다른거 먹자</h2>
      <div class="sidebar__button-wrapper">
        <button class="sidebar__button">맥도날드</button>
        <button class="sidebar__button">버거킹</button>
      </div>
    </aside>
```
BEM에서 '--'와 -'__'를 쓰는 이유가 클래스 이름을 케밥 케이스로 정의하는 케이스가 있어서라고 해요! 
케밥케이스로 이름을 정의하는 게 아니면 '__'나 '--'로 요소를 구분하면 좋을 것 같아요